crates/aptos/src/genesis/mod.rs

```diff
@@ -283,6 +283,15 @@ fn get_config(client: &Client, user: &str) -> CliTypedResult<ValidatorConfigurat
     )?
     .unwrap_or_default();
 
+    // Default to true for whether the validator should be joining during genesis.
+    let join_during_genesis = parse_optional_option(
+        &owner_config.join_during_genesis,
+        owner_file,
+        "join_during_genesis",
+        bool::from_str,
+    )?
+    .unwrap_or(true);
+
```

Comment:
Glad this turned out to be so straightforward.

Do we want it to be defaulted to true or to false?

Reply:
True for now to maintain the previous default behavior.